README.md

```diff
@@ -28,7 +28,12 @@ In your react-native app...
 Make sure to install all peer dependencies:
 
 ```
-npm install --save @photon-sdk/photon-lib react-native-randombytes react-native-keychain @photon-sdk/react-native-icloudstore @react-native-community/async-storage @photon-sdk/react-native-tcp @react-native-community/google-signin react-native-google-drive-api-wrapper node-libs-react-native
+npm install --save @photon-sdk/photon-lib react-native-randombytes react-native-keychain @photon-sdk/react-native-icloudstore @react-native-async-storage/async-storage @photon-sdk/react-native-tcp @react-native-google-signin/google-signin @robinbobin/react-native-google-drive-api-wrapper node-libs-react-native
+
+OR
+
+yarn add @photon-sdk/photon-lib react-native-randombytes react-native-keychain @photon-sdk/react-native-icloudstore @react-native-async-storage/async-storage @photon-sdk/react-native-tcp @react-native-google-signin/google-signin @robinbobin/react-native-google-drive-api-wrapper node-libs-react-native
+
```

Comment:
Does yarn use the `package.json` too or would it create its own lock file? I wonder if that would create compatibility issues...

Reply:
Yarn will crate its own lock file.

Reply:
Usually projects support on lock file format. Documenting yarn here seems confusing as we don't actually commit the yarn lock file. And committing both lock files would create compatibility nightmares. Which is why I'd suggest removing the yarn part in the readme. If you feel strongly about yarn and you'd like to migrate photon to use yarn instead, might I suggest doing it for all git repos in a separate PR?